tests/topotests/lib/common_config.py

```diff
@@ -2025,6 +2031,51 @@ def shutdown_bringup_interface(tgen, dut, intf_name, ifaceaction=False):
     interface_set_status(router_list[dut], intf_name, ifaceaction)
 
 
+def stop_router(tgen, router):
+    """
+    Router"s current config would be saved to /etc/frr/ for each deamon
```

Comment:
hmm, is this really what's happening? that sounds like several potential problems: a) this can only be used for a single router in the entire running suite, but it doesn't prevent itself from being misused, and b) it will make changes to the actual host system's frr installation, which might also have consequences.

Reply:
There was a problem with doc string. It is corrected now. Correct path is /tmp/topotest/<suite>/<router>